models/public/deeplabv3/deeplabv3.md

```diff
@@ -0,0 +1,71 @@
+# deeplab-v3
+
+## Use Case and High-Level Description
+
+DeepLab is a state-of-art deep learning model for semantic image segmentation. For details see [paper](https://arxiv.org/pdf/1706.05587.pdf)
+
+## Example
+
+## Specification
+
+| Metric            | Value                |
+|-------------------|----------------------|
+| Type              | Semantic segmentation|
+| GFLOPs            | 11.469               |
+| MParams           | 23.819               |
+| Source framework  | Tensorflow           |
+
+## Accuracy
+
+## Performance
+
+## Input
+
+### Original model
+
+1. Name: `ImageTensor`, shape: [1x513x513x3] - An input image in the format [BxHxWxC],
+   where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order - BGR.
+
+### Converted model
+
+1. Name: `mul_1/placeholder_port_1`, shape: [1x3x513x513] - An input image in the format [BxCxHxW],
```

Comment:
Replace layer name with common information about I/O
(for example, [first](https://github.com/opencv/open_model_zoo/pull/174/files#diff-6029b2afd24a4c6d0c2ed2272012623aR30), [second](https://github.com/opencv/open_model_zoo/pull/174/files#diff-027ad4f9d5c40018de6d92e6938c4a7cR41))